doc/ref/plotting_options/size_layout.ipynb

```diff
@@ -0,0 +1,366 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2ae44fa2-2ef0-4204-b9bb-b6051dfefd7c",
+   "metadata": {},
+   "source": [
+    "# Size And Layout Options\n",
+    "\n",
+    "\n",
+    "```{eval-rst}\n",
+    ".. plotting-options-table:: Size And Layout Options\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c20d146d-c67f-40be-b505-edadc4e94bd7",
+   "metadata": {},
+   "source": [
+    "(option-aspect)=\n",
+    "## `aspect`\n",
+    "\n",
+    "The `aspect` option controls the **frame** shape by setting the ratio of its width to its height. By default (`aspect=None`), HoloViews chooses an appropriate ratio automatically. Use:\n",
+    "\n",
+    "- `'square'`  \n",
```

Comment:
Bit of a weird formatting.

<img width="1053" alt="image" src="https://github.com/user-attachments/assets/1248a4b6-ec73-4b24-8a24-763fc6b59e1d" />

```
- `'square'`: Forces a square frame
  - If you set only ...
  - ...
```

How about?